scene/resources/immediate_mesh.cpp

```diff
@@ -144,6 +144,9 @@ void ImmediateMesh::surface_add_vertex_2d(const Vector2 &p_vertex) {
 
 	active_surface_data.vertex_2d = true;
 }
+
+#define SMALL_VEC3 Vector3(0.00001, 0.00001, 0.00001)
```

Comment:
I thought the same thing, but do I need to change the description in RendererServer as well?

Reply:
Since it will be called every frame, it may be better for performance to pre-define it.
@clayjohn @reduz What do you think?

Reply:
I don't think this has any impact on performance, this is a pre-processor define so the exact code is injected in the source when compiling, so the code is strictly equivalent to using `Vector3(CMP_EPSILON, CMP_EPSILON, CMP_EPSILON)` inline.

If you want to pre-define it for performance, then it can be made a class constant instead of a define.

Reply:
Changed those to local variables.